### ISSUE TITLE
New user issues

### ISSUE THREAD

**Issue by vincentdavis** [we-race-here/wrh-organization#58]
Steps to reproduce issues.

1. Uploaded 2 new users to Colorado Cycling organization _Walker 1 Demo walker1@vdavis.net, Walker 2 Demo walker2@vdavis.net
2. Try to create new user accounts walker1@vdavis.net, ... **This gives an error**, see screenshot 1 Trying to create again indicates the user exists already.
3. I **can not login** with the new user, not active, I never get an email. I went to admin and activated the user. Now I can login.
4. Now that I am logged in, I go to my Organizations and I see a full list of orgs, I am expecting to see no orgs or maybe Colorado cycling since a matching email exisits.
5. When I go to my profile, it is blank.

<img width="496" alt="Screen Shot 2022-08-31 at 5 57 14 PM" src="https://user-images.githubusercontent.com/232564/187815841-2f0a518a-4ab2-407d-bfa2-3719fb1d24f1.png">
<img width="550" alt="Screen Shot 2022-08-31 at 5 57 23 PM" src="https://user-images.githubusercontent.com/232564/187815848-efab38f0-e49b-4e35-8457-c427e4bab015.png">
<img width="526" alt="Screen Shot 2022-08-31 at 5 57 48 PM" src="https://user-images.githubusercontent.com/232564/187815852-0a3ab1c7-2a69-4800-b64c-f5e58c3c9adb.png">
<img width="819" alt="Screen Shot 2022-08-31 at 6 03 03 PM" src="https://user-images.githubusercontent.com/232564/187815853-5a908b1b-ac17-486b-a49a-28cc80f65d41.png">
<img width="956" alt="Screen Shot 2022-08-31 at 6 03 17 PM" src="https://user-images.githubusercontent.com/232564/187815855-835ee3b3-0c0e-4476-b2cb-163b2d3dcb07.png">
<img width="909" alt="Screen Shot 2022-08-31 at 6 03 54 PM" src="https://user-images.githubusercontent.com/232564/187815859-e2efd7f8-e4bf-4b8f-958d-2849bf720952.png">

**Comment by pprolancer** [comment]
Hi @vincentdavis 
1- error in step-2 is because of the sendgrid api error. seems the api key is expired or invalid
2- with only activate a user, we cannot activate rider profile account. then I have added a new action in Admin panel to allow admin user to "activate user and rider profile".
3- I have made some updates to show better errors when a user has not registered as rider member.

all above errors should not be faced in normal registration.
the problem was because the user was not activated using email activation link. (because user didn't received email).
then to fix all issues, we have to fix sendgrid api key config.
